### PR TITLE
[PR4:7] test: fix e2e tests with new cobra commands

### LINKED ISSUE #72: test: Ensure working E2E test after cobra-integrations
While waiting for all cobra-integrations to complete their implementations, we've agreed to outcomment the current E2E file(s) in the cobra-branches to avoid failing a now outdated test.

Branches to outcomment E2E...
- [x] [Cache](https://github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/tree/feat_cache_flag_cobra)
- [x] [Fetch](https://github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/tree/feat_fetch_flag_cobra)
- [x] [Generate](https://github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/tree/feat_generate_flag_cobra)
- [x] [Merge](https://github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/tree/feat_merge_flag_cobra)

### PR DESCRIPTION
**Pull request by Slug-Boi**
> Summary:
> 
> Fetcher.go now does not delete the cache-folder every second time we run go test.
> 
> Merged the Cobra-integration branches in here.
> 
> The E2E-test has been changed so it now uses the Cobra-flags.
> 
> The E2E now passes.
> - Haruyzal

Resolves #72

### CODE REVIEW

File: src/Fetcher/fetcher_test.go [hunk 1]
```diff
@@ -59,13 +62,13 @@ func TestFetchFilesValidArguments(t *testing.T) {
...
-		"web,hold,language,false,0,Lorem ipsum dolor sit amet consectetur adipiscing elit.",
-		"react,hold,language,false,0,Lorem ipsum dolor sit amet consectetur adipiscing elit."}
+	"Python,hold,language,false,0,Lorem ipsum dolor sit amet consectetur adipiscing elit.",
+	"web,hold,language,false,0,Lorem ipsum dolor sit amet consectetur adipiscing elit.",
+	"react,hold,language,false,0,Lorem ipsum dolor sit amet consectetur adipiscing elit."}
 
 	for i := range expected_lines {
 		if !(expected_lines[i] == template_lines[i]) {
-			t.Errorf("Mismatch in downloaded file. Expected: %v \n Retrieved: %v", expected_lines[i], template_lines[i])
+			t.Errorf("Mismatch in downloaded file. Expected: %v \n Retrieved: %v",expected_lines[i], template_lines[i])
```
Comment (lasselundstenjensen) [inline src/Fetcher/fetcher_test.go:71]: Nitpick: Someone did not run gofmt :)

File: test/end2end_test.go [hunk 2]
```diff
@@ -39,52 +50,32 @@ func TestEndToEnd(t *testing.T) {
...
-			<li>Is new: false</li>
-			<li>Moved: 0</li>
-			<li>Desc: Also a description</li>
+	// Slices for assertion/checking with contains
+	correctBlipNames := []string{
+		"TestBlip1",
+		"TestBlip2",
+		"TestBlip3",
+		"TestBlip4",
+	}
```
Comment (lasselundstenjensen) [inline test/end2end_test.go:66]: Assume this is temporary for tests?

File: test/end2end_test.go [hunk 3]
```diff
@@ -39,52 +50,32 @@ func TestEndToEnd(t *testing.T) {
...
-			<li>Is new: false</li>
-			<li>Moved: 0</li>
-			<li>Desc: Also a description</li>
+	// Check if index DOES NOT contain some of the Blips' names.
+	for _, name := range correctBlipNames {
+		if !strings.Contains(string(indexHTMLContent), name) {
+			t.Errorf("Expected Blip-name %q not found in index.html", name)
+		}
+	}
+}
```
Comment (lasselundstenjensen) [inline test/end2end_test.go:80]: Same as previous comment.

**Review by lasselundstenjensen — APPROVED** [review]

File: test/end2end_test.go (hunk 2 above)
Comment (August-Brandt) [reply test/end2end_test.go:66]: Those are names of blips from the generated test files that are used to test that merging works

**Pull request merged**